roadmap/implementors-guide/src/node/backing/candidate-backing.md

```diff
@@ -39,36 +50,37 @@ The subsystem should maintain a set of handles to Candidate Backing Jobs that ar
 
 The Candidate Backing Job represents the work a node does for backing candidates with respect to a particular relay-parent.
 
-The goal of a Candidate Backing Job is to produce as many backable candidates as possible. This is done via signed [`Statement`s](../../types/backing.md#statement-type) by validators. If a candidate receives a majority of supporting Statements from the Parachain Validators currently assigned, then that candidate is considered backable.
+The goal of a Candidate Backing Job is to produce as many backable candidates as possible. This is done via signed [`Statement`s][STMT] by validators. If a candidate receives a majority of supporting Statements from the Parachain Validators currently assigned, then that candidate is considered backable.
 
 ### On Startup
 
-* Fetch current validator set, validator -> parachain assignments from runtime API.
+* Fetch current validator set, validator -> parachain assignments from [`Runtime API`][RA] subsystem using [`RuntimeApiRequest::Validators`][RAM] and [`RuntimeApiRequest::DutyRoster`][RAM]
 * Determine if the node controls a key in the current validator set. Call this the local key if so.
-* If the local key exists, extract the parachain head and validation function for the parachain the local key is assigned to.
+* If the local key exists, extract the parachain head and validation function from the [`Runtime API`][RA] for the parachain the local key is assigned to by issuing a [`RuntimeApiRequest::Validators`][RAM]
+* Issue a [`RuntimeApiRequest::SigningContext`][RAM] message to get a context that will later be used upon signing.
 
-### On Receiving New Signed Statement
+### On Receiving New Candidate Backing Message
 
-```rust
-if let Statement::Seconded(candidate) = signed.statement {
-  if candidate is unknown and in local assignment {
-    spawn_validation_work(candidate, parachain head, validation function)
+```rust,ignore
+match msg {
+  CandidateBackingMessage::Second(hash, candidate) => {
+    if candidate is unknown and in local assignment {
+      spawn_validation_work(candidate, parachain head, validation function)
+    }
   }
 }
-
-// add `Seconded` statements and `Valid` statements to a quorum. If quorum reaches validator-group
-// majority, send a `BlockAuthorshipProvisioning::BackableCandidate(relay_parent, Candidate, Backing)` message.
 ```
 
-### Spawning Validation Work
+Add `Seconded` statements and `Valid` statements to a quorum. If quorum reaches validator-group majority, send a [`ProvisionerMessage::ProvisionableData(ProvisionableData::BackedCandidate(BackedCandidate))`][PM] message.
+
+### Validating Candidates.
 
-```rust
+```rust,ignore
```

Comment:
The problem I have with these `,ignore` blocks is that it turns off syntax highlighting in my editor for some reason